components/organisms/Layout.js

```diff
@@ -1,70 +1,100 @@
 import PropTypes from "prop-types";
 import Head from "next/head";
-import { ActionButton } from "../atoms/ActionButton";
-import { SearchBar } from "../atoms/SearchBar";
 import { Banner } from "../atoms/Banner";
+import { Footer } from "../organisms/Footer";
+import { Header } from "../organisms/Header";
 
 /**
  * Component which defines the layout of the page for all screen sizes
  */
-export const Layout = ({
-  bannerText,
-  bannerTitle,
-  children,
-  bannerStartTestingText,
-  bannerBecomeTesterText,
-}) => {
+export const Layout = ({ bannerText, bannerTitle, children }) => {
   return (
     <div className="overflow-x-hidden">
       <header>
-        <div className="layout-container">
-          <a href="#">
-            <img src="#" alt="Governement of Canada" />
-            <span lang="en">Government of Canada </span>
-            <span lang="fr">Gouvernment du Canada</span>
-          </a>
-        </div>
-
-        <div className="layout-container">
-          <a href="#">Français</a>
-          <SearchBar placeholder={"Search Canada.ca"} dataCy={"search-bar"} />
+        <div className="layout-container ">
+          <Header
+            headerLogoAltText="Symbol of the Government of Canada"
+            headerLogoImage="/sig-blk-en.svg"
+          />
         </div>
-
+        <div className="mb-2 border-t pb-2 mt-4"></div>
         <nav className="layout-container">Menu</nav>
         {bannerText && bannerTitle ? (
           <Banner siteTitle={bannerTitle} headline={bannerText} />
         ) : null}
       </header>
-      <main>{children}</main>
-      <footer className="layout-container mt-10">
-        <div>
-          <p>
-            Experiencing an issue with this product or have you spotted an
-            error?
-          </p>
-          <ActionButton
-            text={"Report a problem"}
-            secondary
-            dataCyButton={"report-problem-button"}
-          />
+      <main>
+        <div className="layout-container mt-10 mb-4">
+          <div>{children}</div>
         </div>
```

Comment:
Let's remove this `div` around the `main`. It adds too much padding. We are already doing layout container in the child components. 

In this screenshot, you can see the grey bar with the two buttons is not hitting the edge of the screen and the content is pushed too far inwards.

<img width="1067" alt="Screen Shot 2021-04-09 at 11 16 04" src="https://user-images.githubusercontent.com/2454380/114202256-01532d00-9925-11eb-99d1-f3f92b4bf550.png">